ros2controlcli/README.md

```diff
@@ -0,0 +1,146 @@
+# ros2controlcli
```

Comment:
The hidden nodes things is inherited from `ros2node` `NodeNameCompleter`, I don't think a hidden controller manager makes much sense, but it's there to avoid duplicating much of the work. Maybe on the future we can make a PR to allow this argument to be optional on `ros2node`.